sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/entity_resolution/Commons.scala

```diff
@@ -90,10 +97,10 @@ class EntityResolution_RDFData_CountVectorizer(spark: SparkSession, triplesRDD1:
       
     })  
     
-    val filtered_triples = distinct_triples.subtract(remove_triples)
+    val filteredTriples = distinctTriples.subtract(removeTriples)
     
     //Define entity profiles <subject, predicate1:object1 , predicate2:object2 , ... , predicaten:objectn>
-    val entity = filtered_triples.filter(f => (f.getObject().isURI() || f.getObject.getLiteralLanguage == "en" || f.getObject.getLiteralLanguage == ""))
+    val entity = filteredTriples.filter(f => (f.getObject().isURI() || f.getObject.getLiteralLanguage == "en" || f.getObject.getLiteralLanguage == ""))
       .map(f => {
         val key = f.getSubject.getURI.split("/").last.trim()
```

Comment:
What are you trying to get here? The Predicat Name without namespace? (e.g. `http://xmlns.com/foaf/spec/#name` you want to get just `name`, right?) . If so, this is possible via `getLocalName()) from the Jena Node.

Reply:
Resolved.